rclpy/src/rclpy/time_point.cpp

```diff
@@ -0,0 +1,81 @@
+// Copyright 2021 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Include pybind11 before rclpy_common/handle.h includes Python.h
+#include <pybind11/pybind11.h>
+
+#include <rcl/rcl.h>
+
+#include <cstring>
```

Comment:
What is this needed for?

Reply:
It's for [`std::strcmp`](https://github.com/ros2/rclpy/pull/699/files/e8fb88a5b29a07137badef80d28dfebf033e8cbf#diff-2e12c9701227e94447565daeac8ceef7b9e5997b09b09617e1d3c4bcb162c910R75)